packages/client/src/object/fetchPage.ts

```diff
@@ -93,28 +93,34 @@ async function fetchInterfacePage<
   args: FetchPageArgs<Q, L, R, any, S>,
   objectSet: ObjectSet,
 ): Promise<FetchPageResult<Q, L, R, S>> {
-  const result = await OntologiesV2.OntologyInterfaces
-    .search(
-      addUserAgentAndRequestContextHeaders(client, interfaceType),
-      await client.ontologyRid,
+  if (args.$__EXPERIMENTAL_useUnstableInterfaces) {
+    // Use new unstable APIs
```

Comment:
Basically just branching to the right API based on what the flag is set to. This way anyone using existing interface stuff wouldn't be affected and since its an optional param in the fetch page it won't require any code changes either